content/release-10-17-0/streaming-analytics-10-17-0-bundle/10_18_0_2.md

```diff
@@ -0,0 +1,61 @@
+---
+weight: 38
+title: Release 10.18.0.2
+layout: redirect
+---
+
+This release upgrades the Apama-ctrl microservice to use Apama 10.15.3.1 (which is the same as Apama 10.15 Fix 12).
+
+### Fixes
+
+<table>
+<colgroup>
+    <col style="width: 15%;">
+    <col style="width: 70%;">
+    <col style="width: 15%;">
+</colgroup>
+<thead>
+<tr>
+<th style="text-align:left">Component</th>
+<th style="text-align:left">Description</th>
+<th style="text-align:left">Issue</th>
+</tr>
+</thead>
+<tbody>
+
+
+<tr>
+<td style="text-align:left">Analytics Builder</td>
+<td style="text-align:left">The <b>Expression</b> block now calculates the remainder of integer division, as already documented.</td>
```

Comment:
I changed the readme on PAB-4096 (the parent of PAB-4110) to add: 
 Previously the functionality was not implemented and raised a runtime error if defined.